evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -528,76 +472,75 @@ def test_make_pipeline_numpy_input(problem_type):
-                expected_components = [DropNullColumns, Imputer] + estimator_components
+                expected_components = (
+                    [DropNullColumns, Imputer] + standard_scaler + [estimator_class]
+                )
             else:
                 expected_components = (
-                    [DropNullColumns, Imputer] + delayed_features + estimator_components
+                    [DropNullColumns, Imputer]
+                    + delayed_features
+                    + standard_scaler
+                    + [estimator_class]
                 )
-            pipeline.component_graph.compute_order == [
+            assert pipeline.component_graph.compute_order == [
                 component.name for component in expected_components
             ]
 
 
 @pytest.mark.parametrize("input_type", ["pd", "ww"])
 @pytest.mark.parametrize("problem_type", ProblemTypes.all_problem_types)
-def test_make_pipeline_datetime_no_categorical(input_type, problem_type):
-    X = pd.DataFrame(
-        {
-            "numerical": [1, 2, 3, 1, 2],
-            "some dates": pd.date_range("2000-02-03", periods=5, freq="W"),
-        }
+def test_make_pipeline_datetime_no_categorical(
+    input_type, problem_type, get_test_data_from_configuration
+):
+    X, y = get_test_data_from_configuration(
+        input_type, problem_type, column_names=["dates", "numerical"]
     )
-    y = pd.Series([0, 1, 1, 0, 0])
-    if input_type == "ww":
-        X.ww.init()
-        y = ww.init_series(y)
-
     estimators = get_estimators(problem_type=problem_type)
     pipeline_class = _get_pipeline_base_class(problem_type)
-    if problem_type == ProblemTypes.MULTICLASS:
-        y = pd.Series([0, 2, 1, 2])
-
     for estimator_class in estimators:
         if problem_type in estimator_class.supported_problem_types:
             parameters = {}
             if is_time_series(problem_type):
                 parameters = {
-                    "pipeline": {"date_index": "soem dates", "gap": 1, "max_delay": 1},
+                    "pipeline": {"date_index": "dates", "gap": 1, "max_delay": 1},
                     "Time Series Baseline Estimator": {
-                        "date_index": "some dates",
+                        "date_index": "dates",
                         "gap": 1,
                         "max_delay": 1,
                     },
                 }
 
             pipeline = make_pipeline(X, y, estimator_class, problem_type, parameters)
             assert isinstance(pipeline, pipeline_class)
-            delayed_features = []
-            if is_time_series(problem_type):
-                delayed_features = [DelayedFeatureTransformer]
-            if estimator_class.model_family == ModelFamily.LINEAR_MODEL:
-                estimator_components = [StandardScaler, estimator_class]
-            elif estimator_class.model_family == ModelFamily.CATBOOST:
-                estimator_components = [estimator_class]
-            else:
```

Comment:
I found myself being confused by the different if/else statements (some of which are related, others which are not!) so I thought this would be a clearer breakdown but let me know if you have thoughts/opinions!

Reply:
Hot take: We shouldn't support plain numpy arrays anymore.